osc-staging.py

```diff
@@ -331,14 +331,19 @@ def do_staging(self, subcmd, opts, *args):
 
             if version_openqa == version_totest and not totest_dirty:
                 cmd = AcceptCommand(api)
+                accepted_prjs = []
                 for prj in args[1:]:
-                    if not cmd.perform(api.prj_from_letter(prj), opts.force):
+                    if cmd.perform(api.prj_from_letter(prj), opts.force):
+                        accepted_prjs.append(prj)
```

Comment:
I'd likely no go through the indirection here, but do the ```cmd.reset_rebuild_data(accepted_prjs)``` right away for the accepted project.

Consider this (unfortunately still very common) issue: ```osc staging accept C D E F G``` => while accepting G, a failure happens (obs timeout, as so often) -> C D E and F will not be reset in the state; while the person accepting stagings will likely force-accept the remainder of G